tests/e2e/vLLM/configs/w8a8_dynamic_asym.yaml

```diff
@@ -0,0 +1,6 @@
+cadence: "nightly"
+test_type: "regression"
+model: TinyLlama/TinyLlama-1.1B-Chat-v1.0
+scheme: W8A8
+activation_scheme: dynamic_asymmetric
```

Comment:
Please add a model saving path i.e `save_dir`
We also don't need a scheme or activation scheme here.